### PR TITLE
Salt and Pepper noise removal

### PR DESCRIPTION
**Pull request by Dulyaaa**
## Description
The images can contain Gaussian, salt (white dots) and pepper (black dots). To remove them we use different image processing techniques. The following images are few examples.
image
image

### CODE REVIEW

**Comment by maulanaakbardj** [comment]
Hi @Dulyaaa, please use format for python project is .py don't send your notebook(.ipynb), you can make it simpler by creating python function in your project

**Comment by Dulyaaa** [comment]
Hi @maulanaakbardj, updated with python code.

**Comment by maulanaakbardj** [comment]
your code is invalid, please check your code there is an error and it is not clear

File: Python/Salt n pepper noise removal/salt n pepper.py [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+# In[2]:
+
+
+import cv2
+import numpy as np
+import matplotlib.pyplot as plt
+
+# get image
+img = cv2.imread("gaussian Noise.png", 0)
```
Comment (maulanaakbardj) [inline Python/Salt n pepper noise removal/salt n pepper.py:12]: no file

File: Python/Salt n pepper noise removal/salt n pepper.py [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+# In[9]:
+
+
+import cv2
+import numpy as np
+import matplotlib.pyplot as plt
+
+# load image
+img = cv2.imread("gaussian Noise.png", 0)
```
Comment (maulanaakbardj) [inline Python/Salt n pepper noise removal/salt n pepper.py:35]: no file

File: Python/Salt n pepper noise removal/salt n pepper.py [hunk 3]
```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
```
Comment (maulanaakbardj) [inline Python/Salt n pepper noise removal/salt n pepper.py:1]: full code is not clear

File: Python/Salt n pepper noise removal/salt n pepper.py [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+
+# In[17]:
+
+
+import cv2
+import numpy as np
+import matplotlib.pyplot as plt
+
+# get image
+img1 = cv2.imread("saltandpepper1.png", 0)
```
Comment (maulanaakbardj) [inline Python/Salt n pepper noise removal/salt n pepper.py:65]: no file

File: Python/Salt n pepper noise removal/salt n pepper.py [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+img2 = cv2.imread("gaussian Noise.png", 0)
```
Comment (maulanaakbardj) [inline Python/Salt n pepper noise removal/salt n pepper.py:66]: no file

File: Python/Salt n pepper noise removal/salt n pepper.py [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+
+# In[ ]:
+
+
+import cv2
+import numpy as np
+import matplotlib.pyplot as plt
+
+# get image
+img_hubble = cv2.imread("hubble.png", 0)
```
Comment (maulanaakbardj) [inline Python/Salt n pepper noise removal/salt n pepper.py:99]: no file

**Review by maulanaakbardj — COMMENTED** [review]
your code is not clear

File: Python/Salt n pepper noise removal/salt n pepper.py (hunk 1 above)
Comment (Dulyaaa) [reply Python/Salt n pepper noise removal/salt n pepper.py:12]: Did you check my PR fully? I have already added these files.

File: Python/Salt n pepper noise removal/salt n pepper.py (hunk 2 above)
Comment (Dulyaaa) [reply Python/Salt n pepper noise removal/salt n pepper.py:35]: Did you check my PR fully? I have already added these files.

File: Python/Salt n pepper noise removal/salt n pepper.py (hunk 5 above)
Comment (Dulyaaa) [reply Python/Salt n pepper noise removal/salt n pepper.py:66]: Did you check my PR fully? I have already added these files.

File: Python/Salt n pepper noise removal/salt n pepper.py (hunk 4 above)
Comment (Dulyaaa) [reply Python/Salt n pepper noise removal/salt n pepper.py:65]: Did you check my PR fully? I have already added these files.

File: Python/Salt n pepper noise removal/salt n pepper.py (hunk 6 above)
Comment (Dulyaaa) [reply Python/Salt n pepper noise removal/salt n pepper.py:99]: Did you check my PR fully? I have already added these files.

File: Python/Salt n pepper noise removal/salt n pepper.py (hunk 3 above)
Comment (Dulyaaa) [reply Python/Salt n pepper noise removal/salt n pepper.py:1]: Which part is not clear for you @maulanaakbardj.

**Pull request closed without merging**